frontend/packages/local-storage-operator-plugin/src/constants/index.ts

```diff
@@ -21,3 +21,8 @@ export const DISCOVERY_CR_NAME = 'auto-discover-devices';
 export const LOCAL_STORAGE_NAMESPACE = 'openshift-local-storage';
 export const HOSTNAME_LABEL_KEY = 'kubernetes.io/hostname';
 export const LABEL_OPERATOR = 'In';
+
+export const dropdownUnits = {
```

Comment:
We already have `dropdownUnits` here https://github.com/openshift/console/blob/master/frontend/public/components/storage/shared.ts#L35. We can make use of them directly.

Reply:
@a2batic If you look closely, its the other way, i.e.  `Mi` is mapped to `MiB` not `MiB` to `Mi`